src/main/java/com/wcc/platform/repository/postgres/PostgresHeroSectionRepository.java

```diff
@@ -0,0 +1,80 @@
+package com.wcc.platform.repository.postgres;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.wcc.platform.domain.cms.attributes.HeroSection;
+import com.wcc.platform.domain.cms.attributes.Image;
+import com.wcc.platform.domain.cms.attributes.style.CustomStyle;
+import com.wcc.platform.repository.HeroSectionRepository;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Optional;
+import lombok.AllArgsConstructor;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@AllArgsConstructor
+public class PostgresHeroSectionRepository implements HeroSectionRepository {
```

Comment:
we don't need another repository for this, we are using generic pageRepository to take care of that.